e2e/tests/06_node_get.bats

```diff
@@ -20,10 +18,45 @@ teardown() {
 }
 
 @test "Nodes retrieval via kubectl" {
+  poll_until_equals "no nodes" "" "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl get node capsule-control-plane --output=name" 3 5
+
+  kubectl patch tenants.capsule.clastix.io node --type=json -p '[{"op": "add", "path": "/metadata/annotations", "value": {"capsule.clastix.io/enable-node-listing": "true"}}]'
   local list="node/capsule-control-plane"
-  poll_until_equals "node retrieval" "$list" "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl get node capsule-control-plane --output=name" 3 5
+  poll_until_equals "nodes retrieval" "$list" "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl get node capsule-control-plane --output=name" 3 5
 }
 
-@test "Nodes labeling via kubectl" {
+@test "Node labeling via kubectl" {
+  run sh -c "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl label node capsule-control-plane capsule.clastix.io/test=labeling"
+  [ $status -eq 1 ]
+
+  kubectl patch tenants.capsule.clastix.io node --type=json -p '[{"op": "add", "path": "/metadata/annotations", "value": {"capsule.clastix.io/enable-node-listing": "true", "capsule.clastix.io/enable-node-update": "true"}}]'
   poll_until_equals "node labeling" "node/capsule-control-plane labeled" "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl label node capsule-control-plane capsule.clastix.io/test=labeling" 3 5
 }
+
+@test "Node tainting via kubectl" {
+  run sh -c "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl taint nodes capsule-control-plane key1=value1:NoSchedule"
+  [ $status -eq 1 ]
+
+  kubectl patch tenants.capsule.clastix.io node --type=json -p '[{"op": "add", "path": "/metadata/annotations", "value": {"capsule.clastix.io/enable-node-listing": "true", "capsule.clastix.io/enable-node-update": "true"}}]'
+  poll_until_equals "node tainting" "node/capsule-control-plane tainted" "KUBECONFIG=${HACK_DIR}/alice.kubeconfig
+  kubectl taint nodes capsule-control-plane key1=value1:NoSchedule" 3 5
+}
+
+@test "Node cordoning via kubectl" {
+  run sh -c "KUBECONFIG=${HACK_DIR}/alice.kubeconfig kubectl cordon capsule-control-plane"
+  [ $status -eq 1 ]
+
+  kubectl patch tenants.capsule.clastix.io node --type=json -p '[{"op": "add", "path": "/metadata/annotations", "value": {"capsule.clastix.io/enable-node-listing": "true", "capsule.clastix.io/enable-node-update": "true"}}]'
+  poll_until_equals "node cordoning" "node/capsule-control-plane cordoned" "KUBECONFIG=${HACK_DIR}/alice.kubeconfig
+  kubectl cordon capsule-control-plane" 3 5
```

Comment:
Hey, can we __uncordon__ after the execution? Just to restore the initial situation :)

Reply:
fixed